### ISSUE TITLE
Error while parsing

### ISSUE THREAD

**Issue by razagill** [pburtchaell/react-notification#1]
Hi,
I'm using the component and packaging via gulp however I'm getting this error:
```
Parsing file ... /node_modules/react-notification/index.js: Expecting Unicode escape sequence \uXXXX (50:24)
```

**Comment by pburtchaell** [comment]
Hey @razagill, from the README:

> This is a major version zero release. The component is still under development and the API may change at any time.

I'm still working on the component. I will keep this issue open until I finish it.

**Comment by pburtchaell** [comment]
@razagill Just a heads up that I released v1.0.0 today. Checkout the README.

**Comment by razagill** [comment]
Nice, I'll have a look soon.
